### PR TITLE
fix(docs): add `Move` section to README and improve README readability

### PR DESCRIPTION
**Pull request by ndom91**
### [Rendered](https://github.com/ndom91/svelte-interactions/blob/ndom91/readme-cleanup/README.md)

- Add `<summary />` / `<detail />` sections around all sub-sections to improve readability
- Add `Move` docs section

It won't let me tag you as reviewer for whatever reason :thinking: @huntabyte 

Preview screenshot:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +127,7 @@ type PressConfig = PressHandlers {
 
 The `PressConfig` object also includes handlers for all the different `PressHandlers`. These are provided as a convenience, should you prefer to handle the events here rather than the custom `on:press*` events dispatched by the element with the `pressAction`.
 
-Be aware that event if you use these handlers, the custom `on:press*` events for whatever handlers you use will not be dispatched to the element. We only dispatch the events that aren't handled by the `PressHandlers`.
+Be aware that even if you use these handlers, the custom `on:press*` events for whatever handlers you use will not be dispatched to the element. We only dispatch the events that aren't handled by the `PressHandlers`.
```
Comment (huntabyte) [inline README.md]:
```suggestion
Be aware that if you use these handlers, the custom `on:press*` events for whatever handlers you use will not be dispatched to the element. We only dispatch the events that aren't handled by the `PressHandlers`.
```

Comment (huntabyte) [reply README.md]: This was my original mistake but best to get the wording right now 😅

**Review by huntabyte — APPROVED** [review]
Thanks a ton for this!! Will make things much better until I can get some docs stood up for it which I plan to do later this week!

**Comment by ndom91** [comment]
Speaking of docs, what do you like to use for docs sites? Is there a svelte alternative to vitepress, docusaurus, etc? Or you just spin up something custom? 

I've been looking for something for a new svelte project of mine haha

**Comment by huntabyte** [comment]
> Speaking of docs, what do you like to use for docs sites? Is there a svelte alternative to vitepress, docusaurus, etc? Or you just spin up something custom?
> 
> I've been looking for something for a new svelte project of mine haha

There are a few but I've found they don't quite compare to the ones that exist for other frameworks so I have my own which I spin up and modify based on the project. Will likely publish it soon as I get the time to document it but its pretty straightforward. https://github.com/huntabyte/formsnap is using it.

**Review by huntabyte — APPROVED** [review]

**Comment by huntabyte** [comment]
Sorry for the delay in getting this merged, on travel this week! Appreciate you taking the time to do this!

**Pull request merged**